### PR TITLE
[DOCS] Will/docs/how to/metadata stores/expectation on filesystem

### PR DESCRIPTION
**Pull request by Shinnnyshinshin**
Changes proposed in this pull request:
- added documentation to `how_to/metadata_stores/expectation_on_filesystem`
- added recommendation about storing Expectations on git

### CODE REVIEW

**Review by jcampbell — CHANGES_REQUESTED** [review]
I think I see two different versions of this in #1530 and #1531...can you clarify which is correct to review (I had already looked at 1531)

**Comment by Shinnnyshinshin** [comment]
suggested changes from #1531 are reflected in the most recent push

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst [hunk 1]
```diff
@@ -3,12 +3,87 @@
...
-.. admonition:: Admonition from Mr. Dickens
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+By default, newly profiled Expectations are stored in JSON format in the ``expectations/`` subdirectory of your ``great_expectations`` folder.  This guide will help you configure a new storage location for Expectations on your filesystem.
 
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+    - Configured a :ref:`Data Context <_how_to_guides__configuring_data_contexts__how_to_create_a_new_data_context_with_the_cli>`.
+    - Configured an :ref:`Expectation Suite <_how_to_guides__creating_and_editing_expectations__how_to_create_a_new_expectation_suite_using_the_cli>`.
```
Comment (Shinnnyshinshin) [inline docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst]: The statement : Configured an Expectation Suite is currently linked to the How to guide for creating a expectation suite using the CLI 
 https://docs.greatexpectations.io/en/latest/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_the_cli.html


Another option would be the Expectations document under Core Concepts
https://docs.greatexpectations.io/en/latest/reference/core_concepts.html#expectations

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst [hunk 2]
```diff
@@ -3,12 +3,87 @@
...
 
-.. admonition:: Admonition from Mr. Dickens
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+By default, newly profiled Expectations are stored in JSON format in the ``expectations/`` subdirectory of your ``great_expectations`` folder.  This guide will help you configure a new storage location for Expectations on your filesystem.
 
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+    - Configured a :ref:`Data Context <how_to_guides__configuring_data_contexts__how_to_create_a_new_data_context_with_the_cli>`.
```
Comment (Shinnnyshinshin) [inline docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst]: Configured a Data Context currently links to the How_to_guide for creating a new Data Context suite using the CLI.

https://docs.greatexpectations.io/en/latest/how_to_guides/configuring_data_contexts/how_to_create_a_new_data_context_with_the_cli.html

Another option would be to link it to the Core Concepts article
https://docs.greatexpectations.io/en/latest/reference/core_concepts/data_context.html

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst (hunk 1 above)
Comment (Shinnnyshinshin) [reply docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst]: Resolved by @abegong. This now points to Tutorial 
https://docs.greatexpectations.io/en/latest/tutorials/getting_started/create_your_first_expectations.html

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst (hunk 2 above)
Comment (Shinnnyshinshin) [reply docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst]: resolved by @abegong . Now points to Tutorial 

https://docs.greatexpectations.io/en/latest/tutorials/getting_started/initialize_a_data_context.html

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst [hunk 3]
```diff
@@ -3,16 +3,85 @@
...
+Steps
+-----
+
+1. First create a new folder where you would to store your Expectations, and move your existing Expectation files over to the new location. In our case, the name of the Expectations file is ``npi_expectations`` and the path to our new storage location is ``/shared_expectations``.
+
+.. code-block:: bash
+
+    # in the great_expectations folder
+    mkdir shared_expectations
+    mv expectations/npi_expectations.json /shared_expectations
```
Comment (jcampbell) [inline docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst]:
```suggestion
    mv expectations/npi_expectations.json shared_expectations/
```

**Review by jcampbell — CHANGES_REQUESTED** [review]
Approved, but note one change that I think needs to be made first.

**Comment by Shinnnyshinshin** [comment]
ToDo : Section describing how github can be used to store expectations.

File: docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst [hunk 4]
```diff
@@ -3,16 +3,125 @@
...
+    great_expectations suite list
+
+    1 Expectation Suite found:
+        - npi_expectations
+
+
+Additional Notes
+----------------
+
+- For best practices, we highly recommend that you store Expectations in a version-control system like Git. The JSON format of Expectations will allow for informative diff-statements and effective tracking of modifications. In the example below, 2 changes have been made to ``npi_expectations``.  The Expectation ```expect_table_column_count_to_equal`` was been changed from ``330`` to ``333`` to ``331``.
```
Comment (Shinnnyshinshin) [inline docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem.rst:89]: This is the blurb that recommends storing Expectations on Git. Does it need a `punchy` conclusion?

**Review by jcampbell — APPROVED** [review]

**Pull request merged**